### DISCUSSION TITLE
YoloV8 Transfer learning

### DISCUSSION THREAD

**Discussion by saiarun7** [ultralytics/ultralytics#1734] [Q&A]
Hello, I have been searching for any documentation on yoloV8 transfer learning, but couldn't find any. I tried using resume =last.pt , but theres this issue rising " AssertionError: Resume checkpoint f{last} not found. Please pass a valid checkpoint to resume from, i.e. yolo resume=path/to/last.pt" .is there any way someone can help me this issue?

**Comment by djoiiwalraven** [comment]
I'm not sure what resume does. But I'm pretty sure that if you set model=best.pt you will be able to train your model from your previous best trained weights.

I think here it's also explained what you are looking for:
https://github.com/ultralytics/ultralytics/issues/802#issuecomment-1416931839

**Reply by glenn-jocher** [reply]
@djoiiwalraven hello! For transfer learning with YOLOv8, you typically want to start from a set of pretrained weights and then fine-tune on your custom dataset. The `resume` argument is used to continue training from a specific checkpoint. If you're encountering an error stating that the checkpoint is not found, make sure that the `last.pt` file exists in your working directory or provide the correct path to where the checkpoint is saved.

If you want to start from the best weights of a previous training session, you can set the `weights` argument to `best.pt` when initiating training. This will load the weights from `best.pt` and start training from there.

For more detailed guidance on transfer learning and the various training options available in YOLOv8, please refer to our documentation at https://docs.ultralytics.com. 😊

**Comment by ExtReMLapin** [comment]
you might also want to freeze the backbone

**Reply by DavidZSchneider** [reply]
@ExtReMLapin , How to freeze it?
